source/containers/FormField/FormField.js

```diff
@@ -160,17 +162,9 @@ const FormField = ({
 
   /** Checks if the field is conditional on another input, and if so,
    * evaluates whether this field should be active or not */
-  const checkCondition = (questionId) => {
-    if (!questionId) return true;
-
-    if (typeof questionId === 'string') {
-      if (questionId[0] === '!') {
-        const qId = questionId.slice(1);
-        return !answers[qId];
-      }
-      return answers[questionId];
-    }
-    return true;
+  const checkCondition = (condition) => {
```

Comment:
I understand why this check is done here, but the result of the check is passing all questions down to a single field. It would be nice if we could do this check higher up where we have access to all the questions without passing them down.

Reply:
The reason for this is that a field get a broader scope than just rendering the input field at hand. Or at least that is what my view of the FormField component is, a field renderer.

Reply:
Yeah, I see your point. The check could probably be lifted even to the Form level, so that we filter the questions we send to a step based on such a check. Would take some refactoring, but probably not too much, and then the FormField wouldn't need to know about all questions.